README.md

```diff
@@ -4,6 +4,13 @@
 Backend RESTful API for managing event data to be consumed by clients, like *website-frontend*.  Built with Java and Spring Boot.
 
 ## Project Setup
+Please review the local development environment steps at * link to the section in the wiki * and after you have Java and Maven installed do the following to setup the project.
```

Comment:
Hey @mdiwale , so far so good, but have a couple changes for this section:
1. "* link to the section in the wiki *" was meant to be a prompt to you to include a link to the Onboarding Documentation for that section.  Please update.
1.  Please use numbered listing for the steps, e.g.
> 1.  Step 1 
> 1.  Step 2
3.  Where does this project come from you reference in Step 1?
4. Any steps for installing dependencies?
5. How do I build or run tests?  An example command to validate everything is working might be good.